src/app/components/BibPage/DefinitionList.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/*
+ * DefinitionList
+ * Expects data in the form of [{ term: '', definition: ''}, {...}, ...].
+ */
+const DefinitionList = ({ data }) => {
+  const getDefinitions = (definitions) => {
+    if (!definitions && !definitions.length) {
+      return null;
+    }
+
+    return definitions.map((item) => (
+      [
+        (<dt>{item.term}</dt>),
```

Comment:
Should we return `undefined` or convert it into `''` if no values from `item.term` or `item.definition`